plannerbenchmark/postProcessing/metrics.py

```diff
@@ -40,42 +62,67 @@ def computeMetric(self, data):
 
 
 class TimeToReachGoalMetric(Metric):
+    """Metric to compute the time it took to reach the goal.
+
+    Requires the threshold, `des_distance`.
+    """
+
     def computeMetric(self, data):
-        m = self._params['m']
+        m = self._params["m"]
         fks = np.stack([data[name] for name in self._measNames[:m]]).T
         goal = np.stack([data[name] for name in self._measNames[m:-1]]).T
         t = data[self._measNames[-1]]
```

Comment:
't' can be given a more represnetative name, for example, it can be renamed to `timetoreachgoal` if it is meant to indicate that here.

Reply:
Changed to timesteps in 330157021817521712d970d5ec08765d5f4324bc